.github/ISSUE_TEMPLATE/10_bug_report.yml

```diff
@@ -0,0 +1,71 @@
+name: 🐞 Bug Report
+description: Create a report about something that is not working
+body:
+  - type: markdown
+    attributes:
+      value: |
+        More information on our issue management policies can be found here: https://aka.ms/aspnet/issue-policies
+
+        Please keep in mind that the GitHub issue tracker is not intended as a general support forum, but for reporting **non-security** bugs and feature requests.
+
+        If you believe you have an issue that affects the SECURITY of the platform, please do NOT create an issue and instead email your issue details to secure@microsoft.com. Your report may be eligible for our [bug bounty](https://www.microsoft.com/en-us/msrc/bounty-dot-net-core) but ONLY if it is reported through email.
+        For other types of questions, consider using [Discussions](https://github.com/dotnet/aspire/discussions).
+  - type: checkboxes
+    attributes:
+      label: Is there an existing issue for this?
+      description: Please search to see if an issue already exists for the bug you encountered ([aspnetcore/issues](https://github.com/dotnet/aspire/issues?q=is%3Aissue)). More information on our issue management policies is available [here](https://aka.ms/aspnet/issue-policies).
+      options:
+        - label: I have searched the existing issues
+          required: true
+  - type: textarea
+    attributes:
+      label: Describe the bug
+      description: A clear and concise description of what the bug is.
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Expected Behavior
+      description: A clear and concise description of what you expected to happen.
+    validations:
+      required: false
+  - type: textarea
+    attributes:
+      label: Steps To Reproduce
+      description: |
+        We ❤ code! Point us to a [minimalistic project which reproduces this issue (repro)](https://github.com/dotnet/aspnetcore/blob/main/docs/repro.md) hosted in a public GitHub repo.
+
+        We will close this issue if:
+        - The repro project you share with us is complex. We can't investigate custom projects, so don't point us to such, please.
+        - If we will not be able to repro the behavior you're reporting.
+        - If the repro project is attached as a `.zip` file.
+        - If the GitHub repro project is set to `Private`.
+    validations:
+      required: false
+  - type: textarea
+    attributes:
+      label: Exceptions (if any)
+      description: Include the exception you get when facing this issue.
+      placeholder:
+    validations:
+      required: false
+  - type: input
+    attributes:
+      label: .NET Version info
+      description: |
+        Run `dotnet --info`
```

Comment:
Would getting the `aspire` workload version also be useful?

Reply:
That comes in with the dotnet --info call